PdfiumAndroid/src/main/cpp/CMakeLists.txt

```diff
@@ -57,11 +57,6 @@ find_library( # Sets the name of the path variable.
 target_link_libraries( # Specifies the target library.
         pdfiumandroid
 
-        libabsl
```

Comment:
Why are you changing these dependencies?  Its not necessarily a dead-breaker, but it needs to be clearly justified.

Reply:
Because, libpdfium.so, already contains all required dependencies